XYZDrivers/src/main/webapp/admin.jsp

```diff
@@ -17,11 +17,11 @@
 
               <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
                 <ul class="nav navbar-nav">
-                  <li><a href="#">Home</a></li>
+                  <li><a href="/XYZDrivers">Home</a></li>
                 </ul>
```

Comment:
We don't need this right now, but eventually we'll need a link to either the admin or member page in the navigation bar as per the spec.